polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -1040,26 +1031,3 @@ struct ConvertPolygeistToLLVMPass
   }
 };
 } // namespace
-
-std::unique_ptr<Pass>
-mlir::createConvertPolygeistToLLVMPass(const LowerToLLVMOptions &options) {
-  auto allocLowering = options.allocLowering;
-  // There is no way to provide additional patterns for pass, so
-  // AllocLowering::None will always fail.
-  assert(allocLowering != LowerToLLVMOptions::AllocLowering::None &&
-         "LLVMLoweringPass doesn't support AllocLowering::None");
-  bool useAlignedAlloc =
-      (allocLowering == LowerToLLVMOptions::AllocLowering::AlignedAlloc);
-  return std::make_unique<ConvertPolygeistToLLVMPass>(
-      options.useBarePtrCallConv, false, options.getIndexBitwidth(),
-      useAlignedAlloc, options.dataLayout);
-}
-
-std::unique_ptr<Pass> mlir::createConvertPolygeistToLLVMPass() {
-  // TODO: meaningful arguments to this pass should be specified as
-  // Option<...>'s to the pass in Passes.td. For now, we'll provide some dummy
-  // default values to allow for pass creation.
-  auto dl = llvm::DataLayout("");
-  return std::make_unique<ConvertPolygeistToLLVMPass>(false, false, 64u, false,
-                                                      dl);
-}
```

Comment:
This was messy and is no longer needed (auto generated). Default arguments can be defined as usual in the `.td` file